lametro/templates/search/_search_filter.html

```diff
@@ -21,40 +21,31 @@ <h3 class="accordion-header">
-                    {% if facet_name in 'topics,lines_and_ways,phase,bill_type,project,metro_location,geo_admin_location,significant_date,motion_by,plan_program_policy' %}
+                    {% if facet_name != 'sponsorships' %}
+
                         {% include 'search/_ordered_search_filter_items.html' %}
 
-                    {% elif facet_name == 'sponsorships' %}
-                        {% for name, count in facets.fields.sponsorships %}
+                    {% else %}
 
+                        {% for name, count in facets.fields.sponsorships %}
                             {% if count %}
-                                <li class="small d-flex">
+                            <li class="small d-flex">
+                                <div class="skinny-list">
+                                    <a href="#" class="filter-value" data-param="sponsorships_exact:{{name}}" title="{{ name|title }}">
                                     {% if name in selected_facets.sponsorships %}
-                                        <div class="skinny-list">
-                                            <a href="#" class="filter-value" data-param="sponsorships_exact:{{name}}" title="{{ name|title }}">
-                                                <strong>{{ name | title }}</strong>
-                                            </a>
-                                            <a href="#" class="remove-filter-value btn btn-primary btn-sm" data-param="{{facet_name}}_exact:{{name}}" aria-label="Remove {{name}} filter">
-                                                <i class="fa fa-times" aria-hidden="true"></i>
-                                            </a>
-                                        </div>
+                                        <strong>{{ name | title }}</strong>
+                                        <a href="#" class="remove-filter-value btn btn-primary btn-sm" data-param="{{facet_name}}_exact:{{name}}" aria-label="Remove {{name}} filter">
+                                            <i class="fa fa-times" aria-hidden="true"></i>
+                                        </a>
                                     {% else %}
                                         <a href="#" class="filter-value" data-param="sponsorships_exact:{{name}}" title="{{ name|title }}">
                                             {{ name | title }}
                                         </a>
                                     {% endif %}
-
-                                    <span class="badge badge-facet ms-auto">{{ count }}</span>
-
-                                </li>
-                            {% endif%}
-
-                        {% endfor %}
-                    {% else %}
-
-                        {% for name, count in item_list %}
-                            {% if count %}
-                                {% include 'search/_ordered_search_filter_items.html' %}
+                                    </a>
```

Comment:
Voiceover is reading each facet link in the Meetings filter twice, and it looks like this produces nested links (which then resolves to just two links next to each other).

Removing the outermost anchor tag might be what we're looking for here. Maybe something like:

```html
<div class="skinny-list">
  {% if name in selected_facets.sponsorships %}
      <strong>{{ name | title }}</strong>
      <a href="#" class="remove-filter-value btn btn-primary btn-sm" data-param="{{facet_name}}_exact:{{name}}" aria-label="Remove {{name}} filter">
          <i class="fa fa-times" aria-hidden="true"></i>
      </a>
  {% else %}
      <a href="#" class="filter-value" data-param="sponsorships_exact:{{name}}" title="{{ name|title }}">
          {{ name | title }}
      </a>
  {% endif %}
</div>
```